tests/product/base_product_case.py

```diff
@@ -98,13 +98,13 @@ def detect_presto_rpm(self):
             # are multiple RPMs, the last one is probably the latest
             return sorted(rpm_names)[-1]
         else:
-            # TODO: once the RPM is on Maven Central, pull the RPM from there
-            rpm_filename = 'presto-0.101-1.0.x86_64.rpm'
+            rpm_filename = 'presto-server-rpm.rpm'
```

Comment:
so the rpm name will never have anything else in it?  (x86_64, version number, etc.)